seagulls-space-shooter-demo/src/seagulls/space_shooter/_ship_powered_up_rule.py

```diff
@@ -19,4 +19,4 @@ def check(self) -> None:
             self._state_client.update_state(True)
 
     def _score_threshold_check(self) -> bool:
-        return self._score_tracker.get_score() >= 75
+        return self._score_tracker.get_score() >= 20
```

Comment:
do we want to set this back to 75 before merging?

Reply:
Yep, updated